test/integration/helpers/Npm.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    internal class Npm
+    {
+
+        public static string GetNpmPrefixPath()
+        {
+            string npmPath = GetNpmExecutablePath();
+            string npmPrefixPath = RunCommand(npmPath, "-g root");
+
+            return npmPrefixPath.Trim();
+        }
+
+        private static string RunCommand(string command, string arguments, int timeoutMilliseconds = 30000)
+        {
+            try
+            {
+                using (Process process = new Process
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = command,
+                        Arguments = arguments,
+                        RedirectStandardOutput = true,
+                        RedirectStandardError = true,
+                        UseShellExecute = false,
+                        CreateNoWindow = true,
+                    }
+                })
+                {
+                    process.Start();
+                    string output = process.StandardOutput.ReadToEnd();
+                    string errorOutput = process.StandardError.ReadToEnd();
+                    _ = process.WaitForExit(timeoutMilliseconds);
+
+                    int exitCode = process.ExitCode;
+                    if (exitCode == 1)
+                    {
+                        throw new NpmUnknownCommandException($"Command: `{arguments}` exited with code {exitCode}. Error: {output}");
```

Comment:
I think we may be more explicit about the exception message here if we know that exit code 1 means an invalid npm command

Reply:
It's not always the case when exit code =1.
Could be other issues other than npm related

Reply:
then what is the reason for making exit code 1 so special?

Reply:
I forgot to push the last commit :(

Reply:
I removed the ExitCode != 0 and kept only exitCode == 1 which is the one I saw while testing